tests/test_init_and_backend.py

```diff
@@ -0,0 +1,69 @@
+import os
```

Comment:
SPDX license comment

Reply:
To add a little more context, we require every file to start with an appropriate license comment for legal purposes. Instead of putting the entire license in each file, we put the SPDX license identifier in the file instead which can be used to look up the full license on https://spdx.org/licenses/.

You can simply copy the license comment from another file to this file as well to fix this.